### PR TITLE
⬆️(hawthorn/1/oee) upgrade fonzie to 0.2.1

### PR DESCRIPTION
**Pull request by rmoch**
upgrade fonzie to its latest version which fixes an ACL bug

### CODE REVIEW

File: releases/hawthorn/1/oee/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ release.
 - Add LOCALE_PATHS to configurable settings
 - Remove useless timezone configuration
 - Remove useless development dependencies
+- Upgraded
+  [fonzie](https://github.com/openfun/fonzie)
```
Comment (sampaccoud) [inline releases/hawthorn/1/oee/CHANGELOG.md]: I don't think we should include links in the CHANGELOG. It should be minimal.
```
- Upgraded fonzie to v0.2.1
```

Comment (rmoch) [reply releases/hawthorn/1/oee/CHANGELOG.md]: Well previous changelogs have such links, if I remove this one, should I also remove all of them ?

Comment (sampaccoud) [reply releases/hawthorn/1/oee/CHANGELOG.md]: ok then :man_shrugging:

File: releases/hawthorn/1/oee/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ release.
 - Add LOCALE_PATHS to configurable settings
```
Comment (sampaccoud) [inline releases/hawthorn/1/oee/CHANGELOG.md]: in your commit message, can you remove /release as mentionned by @jmaupetit?

**Pull request merged**